consoles/s3270.pm

```diff
@@ -423,12 +423,11 @@ sub activate ($self) {
     $self->guest_login($bmwqemu::vars{ZVM_PASSWORD} or die 'Need variable ZVM_PASSWORD');
     $self->new_3270_console;
     $self->connect_and_login;
-    return;
+    return;    # uncoverable statement
 }
 
 sub disable ($self) {
     $self->cp_logoff_disconnect();
-    $self->_kill_window();
```

Comment:
If you need this change then please make it explicit with a well name commit message. So far I don't see that

Reply:
I have revert this commit.

Reply:
Brought back this change in a separate new commit.

Reply:
This change is now in a new commit